### ISSUE TITLE
Accessibility: HTML is used to format content & iFrame is missing a title

### ISSUE THREAD

**Issue by kevinkidwell** [cul-it/exhibits-library-cornell-edu#427]
**Description**
These two issues are present for iFrames that are being brought in for various content. 

**Solutions**
Content owners will need to edit the code that is used to embed an iFrame. They must remove any of the following:
1. `frameborder="0"`
2. `cellpadding="x"`
3. `cellspacing="x"`

Content owners must add a title to this embed code that is descriptive of what is being embedded, whether that is a video or other content. They should add the following:
1. `title="description of embedded element"` - replacing the placeholder text within quotes with their own, descriptive text
